moz_telemetry/sandboxes/heka/output/moz_telemetry_s3.lua

```diff
@@ -85,6 +85,10 @@ if experiment_dimensions then
     experiment_batch_dir  = read_config("experiment_batch_dir") or error("experiment_batch_dir must be specified")
     experiment_dimensions = mts3.validate_dimensions(experiment_dimensions)
 end
+local experiment_types = read_config("experiment_types")
+if not experiment_types then experiment_types = {} end
+local experiment_blocklist = read_config("experiment_blocklist")
+if not experiment_blocklist then experiment_types = {} end
```

Comment:
Sets the wrong value when the blocklist is nil/false;

Also this and the above statement allow the config option to be a non-table (which will fail below at run time instead of init time). Something like the following would be more robust:

```lua
local experiment_blocklist = read_config("experiment_blocklist") or {}
assert(type(experiment_blocklist) == "table", "experiment_blocklist must be a table")
```